frontend/tests/services/auth/useUser.test.tsx

```diff
@@ -11,10 +11,10 @@ jest.mock("src/services/fetch/fetchers/clientUserFetcher", () => ({
 }));
 
 const UseUserConsumer = () => {
-  const { error, isLoading, user } = useUser();
+  const { e, isLoading, user } = useUser();
```

Comment:
this is causing tests to fail because the UserProvider returns `error` rather than `e`. I think that's ok, rather than update the userProvider I would revert this change to the test